### PR TITLE
added uses for tin

### PR DESCRIPTION
**Pull request by ThomasMonroe314**
I thought it nice to add more uses for because the only thing tin was used for was bronze and tools that aren't too great.
The texture did not need changing due to the natural tarnishing of the tin.

### CODE REVIEW

File: mods/vessels/init.lua [hunk 1]
```diff
@@ -145,12 +145,12 @@ minetest.register_craft( {
 	}
 })
 
-minetest.register_node("vessels:steel_bottle", {
-	description = "Heavy Steel Bottle (empty)",
+minetest.register_node("vessels:tin_jug", {
+	description = "Durable Tin Jug (empty)",
```
Comment (ThomasMonroe314) [inline mods/vessels/init.lua]: I didnt use heavy here because it didnt sound right

**Comment by C1ffisme** [comment]
Heavy Steel Bottles need to be aliased to Tin Jugs.

**Comment by ThomasMonroe314** [comment]
where would i do that?

**Comment by ThomasMonroe314** [comment]
OK so i am re-adding the steel_bottle, but with a different texture.
I am also going to make another pull request for buckets in a little bit

**Comment by paramat** [comment]
I disagree with removing steel bottle, there's no reason to and aliases are best avoided.
The idea of a tin container for water is not good as tin is not a safe and healthy metal for containing food, steel and glass are.
However besides these, i also disagree with adding more stuff to vessels, some of us would like to remove it completely, and we might do so, because it is a specialist homedecor thing that seems out of place in MTG.

The intention of adding more uses for tin is good, but i have to :-1:  sorry.

**Comment by tenplus1** [comment]
Why not alter steel bottle to be metal bottle and allow players to craft it using steel or tin ?

**Comment by ThomasMonroe314** [comment]
> 

> some of us would like to remove it completely, and we might do so, because it is a specialist home-decor thing that seems out of place in MTG. A jug seems pretty useless, it can't even be used as a sealed portable container for water.

@paramat I partially agree with you there, however i was planning on seeing if i could make them work like the buckets mod., or add default drinks. It does seem sort of decorish(if thats even a word), but it also makes MTG presentable as a default game.

As a part of this addition to the uses of tin, I was going to add a tin bucket, maybe with some other buckets.

@tenplus1 I could do that, but if vessels is getting removed it would be useless.

**Comment by tenplus1** [comment]
I would advise against removing vessels mod, it has uses within other mods for crafting recipe's and not everyone wants to install it manually...  Let it be and add to it.

**Comment by paramat** [comment]
Well, uses in other mods is not a reason to keep it, many mods have uses in other mods.
If someone needs it they will have to install it, whether they want to or not is irrelevant, that's just laziness, it only takes a few seconds.

**Comment by C1ffisme** [comment]
@paramat 
> Well, uses in other mods is not a reason to keep it

_Hold on..._

In #1800 you gave a use for copper:

> Copper is used by many optional mods.

Why is it copper should be kept for it's uses in mods, but vessels shouldn't?

**Comment by paramat** [comment]
Because context.
For vessels, i'm saying that 'uses in other mods' does not prevent us from removing something if removing that something is a good idea.
With copper, apart from making bronze and all bronze items, copper blocks, stairs and slabs, it is also a useful and fundamental resource for other mods, for example for piping or conductive cables.
You have taken some statements out of context and made an oversimplified argument to try to suggest hypocrisy, but there is none.

**Comment by C1ffisme** [comment]
> With copper, apart from making bronze and all bronze items, copper blocks, stairs and slabs, it is also a useful and fundamental resource for other mods, for example for piping or conductive cables.

Any mod with liquids that aren't meant to be kept in dirty buckets (such as food, dye or chemicals) will need vessels.

> You have taken some statements out of context and made an oversimplified argument to try to suggest hypocrisy, but there is none.

(To be fair, I probably did. :P I won't get too worked up about it.)

**Comment by paramat** [comment]
Also a jug is not a bottle.

**Comment by paramat** [comment]
> Why not alter steel bottle to be metal bottle and allow players to craft it using steel or tin ?

I disagree, that's making things more generic, and we don't have a desperate need for more uses for tin, it already has more uses than gold.
This bottle should nor be called a jug.
:-1:

**Comment by C1ffisme** [comment]
@paramat On the contrary, we are in a dire need for more uses for tin. Gold is at least useful on servers to craft keys, but tin is obsolete as soon as a much better tool material is available. (And no, decoration doesn't count, because not everyone is interested in building a rocket ship or a giant skyscraper. Heck, some players probably aren't interested in building at all despite that being the only thing to do in this game...)

**Comment by paramat** [comment]
We have many nodes that are only construction materials, tin is also one of these but it also is used to create bronze, so it has as more uses than many nodes.

**Comment by C1ffisme** [comment]
@paramat A node for construction only is fine...

...as long as it's crafted, and it's source materials have more uses than just decoration.

Bronze, Tin and Copper lose their usefulness as soon as Mese or Diamond is available. I could care less about changing the steel bottle into a tin one, but Tin, Copper and Bronze need uses outside of decoration. At least steel is used for locked items, buckets, minecarts and screwdrivers after better tool materials become available.

**Comment by paramat** [comment]
Yeah i agree more uses for some materials is good, but trying to hard to find a use, as this PR does, is not good. Any use has to be justified by something other than 'to give the material a use'.

**Pull request closed without merging**